datasets/simulated.py

```diff
@@ -5,10 +5,8 @@
 # - skipping import to speed up autocompletion in CLI.
 # - getting requirements info when all dependencies are not installed.
 with safe_import_context() as import_ctx:
-    import numpy as np
-    from sklearn.model_selection import train_test_split
-
     from benchopt.datasets import make_correlated_data
+    import numpy as np

```

Comment:
Import from classical packages are usually put before import from the current package (benchopt).
This is nitpick but help with hierarchical information.